mslice/plotting/plot_window/plot_figure_manager.py

```diff
@@ -28,7 +29,11 @@ def __init__(self, number, current_figs):
         self._current_figs = current_figs
 
         # window instance
-        self.window = PlotWindow(manager=weakref.proxy(self))
+        if not is_gui():  # if in cli only mode
```

Comment:
How about we change `create_qapp` -> `create_qapp_if_required()` and then we can just do 
```
qapp.create_qapp_if_required()
self.window = PlotWindow(manager=weakref.proxy(self))
```
It makes the code a little easier to follow.